samples/net/traffic_class/README.rst

```diff
@@ -0,0 +1,77 @@
+.. _tc-sample:
+
+Traffic Class Sample Application
+################################
+
+Overview
+********
+
+The TC sample application for Zephyr will setup two virtual LAN networks and
+starts to send prioritised UDP packets to peer. The use of VLAN network is
```

Comment:
``prioritized`` (American spelling)

It might be clearer to say:

    send prioritized UDP packets from one VLAN network to the other.